paywall/src/components/checkout/NoWallet.tsx

```diff
@@ -0,0 +1,163 @@
+import styled from 'styled-components'
+import React from 'react'
+import { RoundedLogo } from '../interface/Logo'
+
+import { PaywallConfig } from '../../unlockTypes'
+import Media from '../../theme/media'
+import Svg from '../interface/svg'
+
+interface Props {
+  config: PaywallConfig
+}
+
+export const NoWallet = ({ config }: Props) => {
+  return (
+    <React.Fragment>
+      <Header>
+        <Title>
+          {config.icon && <Logo src={config.icon} />}
+          <UnlockedText>Unlocked</UnlockedText>
+        </Title>
+        <p>
+          To enjoy Forbes Online without any ads you&apos;ll need to use a
```

Comment:
Note: this text will need to become part of the paywall configuration as a follow-up